example/ios/Podfile.lock

```diff
@@ -720,4 +720,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6efcecdc92e5f79e311fb05c1f07868ad3c3f905
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.0
```

Comment:
technically should be controlled by the Gemfile https://github.com/appcues/segment-appcues-react-native/blob/main/example/ios/Gemfile#L4 and `bundle exec pod install` would keep this version consistent.

But, not even the old version is consistent, no big deal here on this example app, just something we can try to clean up in the future.